e2e/api/public/api.js

```diff
@@ -0,0 +1,40 @@
+/* eslint func-names:0 */
```

Comment:
It's a pain for generators too but I guess this should still be enabled. We might however add an `.eslintrc` inside the `e2e` folder which extends the root one